src/client/java/io/wispforest/limelight/impl/ui/LimelightScreen.java

```diff
@@ -88,13 +95,24 @@ protected void build(FlowLayout rootComponent) {
         searchBox.setSelectionEnd(0);
     }
 
+    public void updateResultCounter(ResultsContainerComponent resultsContainer) {
+        if (!Limelight.CONFIG.showResultCounter())
+            return;
+        if (resultsContainer == null)
+            resultsSizeLabel.text(Text.empty());
+        else {
+            resultsSizeLabel.text(Text.literal( Integer.toString(resultsContainer.children().size()) ).withColor(LimelightTheme.current().sourceExtensionColor()));
```

Comment:
*ideally* this would be a separate color method (maybe even a separate color entirely)

Reply:
I can make it a separate method. Though personally, I like it being the same color. If you disagree, ofc feel free to change the value

Reply:
i mean it's mostly more that if I ever add custom theme support it should be clear what the color actually controls, plus somebody might want to edit it specifically
it could be the same method it would just have to be renamed

Reply:
Oh btw, for naming, should the word Color be at the end of all of them? All of them are colors. And if it should be the popupBackground one is missing it

Reply:
yeah, they should all be colors i guess
i'll fix the popup background one later

Reply:
Color added, ready for re-review